funnel/static/translations/hi_IN/messages.json

```diff
@@ -1570,14 +1577,14 @@
       ],
       "Your proposal doesn't have a preview video!": [null, ""],
       "Add preview video link": [null, ""],
-      "Session video": [null, ""],
-      "Proposal video": [null, ""],
+      "Session video": [null, "सेशन की वीडियो"],
+      "Proposal video": [null, "प्रस्ताव का वीडियो"],
       "Status: %(status)s": [null, "स्थिति: %(status)s"],
       "View proposal in schedule": [null, ""],
-      "Change status of proposal": [null, ""],
+      "Change status of proposal": [null, "प्रस्ताव की स्थिति बदलें "],
       "Add session video": [null, "सेशन की वीडियो जोड़ें"],
-      "Add proposal video": [null, ""],
-      "This section is empty.": [null, ""],
+      "Add proposal video": [null, "प्रस्ताव का वीडियो जोड़ें"],
+      "This section is empty.": [null, "ये अनुभाग खाली है."],
```

Comment:
Remove full stop in source and translation. Captions should not end in periods. (However, full sentences do, like when rendered using `render_message`)